boards/WeAct-f411ce/doc.txt

```diff
@@ -0,0 +1,59 @@
+/**
+@defgroup    boards_weact-f411ce WeAct-F411CE board
+@ingroup     boards
+@brief       Support for the WeAct-F411CE Board
+
+## Overview
+
+WeAct-F411CE is a board with the same form-factor as the blue/blackpill,
+but with an STM32F411CEU6 and a USB-C connector.
+
+It is available on sites like AliExpress for less than 4€.
+
+## Hardware
+
+![WeAct-F411CE](https://user-images.githubusercontent.com/1301112/69389644-eb5fb080-0ccc-11ea-8002-67d3db851250.png)
+
+### MCU
+| MCU        | STM32F411CE           |
+|:---------- |:--------------------- |
+| Family     | ARM Cortex-M4F        |
+| Vendor     | ST Microelectronics   |
+| RAM        | 128Kb                 |
+| Flash      | 512Kb                 |
+| Frequency  | up to 100MHz          |
+| FPU        | yes                   |
+| Timers | 8 (2x watchdog, 1 SysTick, 6x 16-bit)    |
+| ADCs       | 1x 12-bit             |
+| UARTs      | 3                     |
+| SPIs       | 5                     |
+| I2Cs       | 3                     |
+| RTC        | 1                     |
+| Vcc        | 2.0V - 3.6V           |
+| Datasheet  | [Datasheet](https://www.st.com/resource/en/datasheet/stm32f411ce.pdf) |
+| Reference Manual | [Reference Manual](https://www.st.com/content/ccc/resource/technical/document/reference_manual/9b/53/39/1c/f7/01/4a/79/DM00119316.pdf/files/DM00119316.pdf/jcr:content/translations/en.DM00119316.pdf) |
+
+## Flashing the device
+The device comes with a bootloader that allows flashing via `dfu-util`.
+
+There are two buttons on the board labeled `BOOT0` and `NRST`.
+
+ - Press and hold down `NRST` to reset the CPU
+ - Press `BOOT0` while keeping `NRST` held down
+ - Release `NRST`, afterwards release `BOOT0`
+
+The board will now show up as `0483:df11` - `STM32 BOOTLOADER` and will accept
+firmware using the DFU protocol.
+
+You can upload your RIOT-firmware by typing
+
+```
+make BOARD=WeAct-f411ce flash
+```
+
```

Comment:
Maybe add something like this:

*Note:* You need to have write permissions to the device. On Linux you could add yourself to the `plugdev` group and store the following as `/etc/udev/rules.d/99-weact-f411ce.rules`:

```
SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="df11", GROUP="plugdev", MODE="660"
```

Reply:
Ok, added.